models/web/scratch/interactions_this_run/redshift_postgres/sources.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 sources:
   - name: atomic
     schema: "{{ var('snowplow__atomic_schema', 'atomic') }}"
-    database: "{{ var('snowplow__database', target.database) }}"
+    database: "{{ var('snowplow__database', target.database) if target.type not in ['databricks', 'spark'] else var('snowplow__atomic_schema', 'atomic') }}"
```

Comment:
Let's use the new variable we just introduced

Reply:
I was meaning to remove it actually, cause I moved it to the redshift_postgres folder that is disabled for everything else.

Reply:
Wait is it removed now or not yet? I'm getting confused by GitHub's UI